README.md

```diff
@@ -41,3 +41,145 @@ Package can then be placed directly under `./charts/<centos|redhat>/<chart-name>
+
+### General notes for Helm charts
+Each helm chart name has to contain file OWNERS. E.g. [OWNERS](./charts/redhat/postgresql-imagestreams/OWNERS)
+
+Each helm chart change, in [templates](./charts/redhat/postgresql-persistent/0.0.1/templates) or [OWNERS](./charts/redhat/postgresql-persistent/OWNERS) file
+has to increase version
+
+### How to test Helm chart
+
+For Helm chart testing you should have a connection to OpenShift 4 cluster and download and install oc binary.
+For access to OpenShift 4 cluster please ask phracek@redhat.com.
+
+#### Create a Helm chart package
+
+Before any testing proposes and deploying Helm chart to OpenShift cluster, you have to create a package by command:
+
+```commandline
+$ helm package <path_to_helm_chart>
+```
+
+E.g. in case of postgresql-persistent Helm chart you see message, like:
+
+```commandline
+$ helm package charts/redhat/postgresql-persistent/0.0.1
+Successfully packaged chart and saved it to: <FULL_PATH_TO_CWD>/helm-charts/postgresql-persistent-0.0.1.tgz
+```
+
+#### Deploy Helm chart to OpenShift
+
+Now let's deploy Helm chart to OpenShift by command:
+
+```commandline
+$ helm install <helm_chart_name> <tar_ball_path>
+```
+
+E.g in case of postgresql-persistent Helm chart you see message, like:
+
+```commandline
+$ helm install postgresql-imagestreams postgresql-imagestreams-0.0.1.tgz
+NAME: postgresql-imagestreams001
+LAST DEPLOYED: Tue Apr  4 10:29:33 2023
+NAMESPACE: pgsql-13
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+To check if Helm chart is really deployd, run command:
+
+```bash
+helm list
+```
+
+The output will be like:
+
+```commandline
+$ helm list
+NAME                      	NAMESPACE	REVISION	UPDATED                              	STATUS  	CHART                        	APP VERSION
+postgresql-imagestreams  	pgsql-13 	1       	2023-04-04 10:29:33.878674 +0200 CEST	deployed	postgresql-imagestreams-0.0.1	0.0.1
+postgresql-persistent     	pgsql-13 	1       	2023-03-28 16:59:56.498783 +0200 CEST	deployed	postgresql-persistent-v0.0.1
+```
+
+### How to verify Helm chart
+
+As soon as a Helm chart is ready and tested let's verify it. The first we have to create a package by command:
+
+```commandline
+$ helm test <helm_chart_name> --logs
+```
+
+The output should be like:
+
+```commandline
+$ helm test postgresql-persistent --logs
+NAME: postgresql-persistent
+LAST DEPLOYED: Tue Mar 28 16:59:56 2023
+NAMESPACE: pgsql-13
+STATUS: deployed
+REVISION: 1
+TEST SUITE:     postgresql-persistent-connection-test
+Last Started:   Tue Apr  4 10:31:44 2023
+Last Completed: Tue Apr  4 10:31:52 2023
+Phase:          Succeeded
+
+POD LOGS: postgresql-persistent-connection-test
+postgresql-testing:5432 - accepting connections
+```
+
+
+Use this command to verify if Helm chart has all proper information in case of Helm chart is already merged.
```

Comment:
In case the Helm chart has already been merged, use the following command to verify if all the required information is contained within the Helm chart.